INSTALL.rst

```diff
@@ -12,20 +12,7 @@ provided with the source.
 Dependencies
 ************
 
-A full list of the Cyclus package dependencies is shown below:
-
-====================   ==================
-Package                Minimum Version
-====================   ==================
-``CMake``                2.8
-``boost``                1.46.1
-``libxml2``              2
-``libxml++``             2.36
-``python``               2.7 or 3.3+
-``sqlite3``              3.7.10
-``HDF5``                 1.8.4
-``Coin-Cbc``             2.5
-====================   ==================
+A full list of the Cyclus package dependencies is listed `here <DEPENDENCIES.rst>`_:
```

Comment:
Have y'all built the website when pointing to these changes? (see https://github.com/cyclus/cyclus.github.com/blob/source/rever.xsh) to make sure the website still builds with these changes?

Reply:
I'm not sure the role of  `rever.xsh` still?  There was a time when we used `rever` for releases, but I don't think that's true anymore?  I guess we should decide what we want to do instead, but I do still think it's best to keep only a single copy of these dependencies to ensure they don't diverge.

Reply:
As far as I can tell, for the website pages for dependencies and install Cyclus still point to this file, which points to the files in this repository and the ones in Cycamore. I think it's meant to keep everything in as few places as possible.

Reply:
Are you sure those files aren't pointed to in the `Makefile` under the `gh-preview` target?  Regardless I think we should be fine since we are pointing to `DEPENDENCIES.rst` which maintains the same name in the Sphinx build.  I will confirm with a local build.

Reply:
I am fairly confident that we do not use `rever` anymore - should we remove it from our project repos? (can be a separate issue, don't want to block this PR any longer).  

That being said... you were right @abachma2 the new links I had were broken in the built website.  I wasn't able to find a trivial way to have the links render nicely in GitHub and by Sphinx so I'll leave them as is for now (with the Sphinx links working correctly).   The README is not pulled into the website build so those links are correctly formatted for use on Github.